src/main/java/net/ftb/log/StreamLogger.java

```diff
@@ -32,40 +34,27 @@
 
     private StreamLogger (InputStream from, LogEntry logInfo) {
         instance = this;
-        this.is = from;
+        this.isr = new BufferedReader(new InputStreamReader(from, Charset.forName("UTF-8")));
```

Comment:
isr -> br (renamed is -> isr when making it an InputStreamReader, forgot to change to br when making it a BufferedReader)